### ISSUE TITLE
builds.knightos.org down

### ISSUE THREAD

**Issue by polarisfm** [KnightOS/KnightOS#365]
It seems that builds.knightos.org is down. Along with that, packages.knightos.org gives a "bad gateway" error, and wiki.knightos.org times out.

**Comment by aviallon** [comment]
Apparently it's normal for wiki.
But I'm not sure for packages or build!

**Comment by polarisfm** [comment]
Hm, alright. If paying for hosting is an issue I have a VPS I could stick this stuff on.

**Comment by polarisfm** [comment]
I would like to add something to my "bug report": packages.knightos.org actually prevents the KnightOS SDK from working properly (example below).

`person@computer:~/knightos/test$` knightos init --platform=TI84p test
Notice: Rebuilding existing environment
Installing SDK...
Installing kernel 0.6.11
Installing packages...
Unable to download manifest for core/kernel-headers. Does it exist?
person@computer:~/knightos/test$ knightos init --platform=TI84p test`

It's trying to grab [this package](https://packages.knightos.org/core/kernel-headers) but fails because of the server being down.

**Comment by SirCmpwn** [comment]
I'm working on redoing how packages.knightos.org is deployed. It'll be solved probably within the week.

**Comment by aviallon** [comment]
@SirCmpwn if you have any trouble hosting it, or if you want redundancy, I have a server unused which could be dedicated to hosting this website. I hope it can be useful to you.

**Comment by SirCmpwn** [comment]
Don't worry, I have plenty of infrastructure.

**Comment by SirCmpwn** [comment]
packages.knightos.org is back up.

**Comment by ijm7** [comment]
Both are down again. Can this be re-opened?

**Comment by SirCmpwn** [comment]
Both are considered obsolete and will not be returning to operation.

**Comment by aviallon** [comment]
Why are they obsolete ? Could you then add a tiny github wiki page explaining how to build and install sdk on common Linux distributions (like Ubuntu and Archlinux - since I saw you made an automated Archlinux build).

**Comment by SirCmpwn** [comment]
https://knightos.org/sdk

**Comment by aviallon** [comment]
Oh ok. Sorry for that, didn't see the link on the mainpage.

**Comment by earboxer** [comment]
Is there a way to get the manifest.json files (without having to host our own https://github.com/KnightOS/packages.knightos.org)?

Or is this what https://github.com/KnightOS/KnightOS/issues/385 is about?

**Comment by pixelherodev** [comment]
Definitely not what that issues is about.

What json files are you referring to? Can you link a page mentioning them?

**Comment by earboxer** [comment]
`knightos init` always gives

> Unable to download manifest for core/init

when I removed that dependency from knightos-sdk, then I was able to compile packages without any dependencies (such as init)
```diff
diff --git a/knightos/templates/assembly/assembly.yaml b/knightos/templates/assembly/assembly.yaml
index 1f42f76..07acff9 100644
--- a/knightos/templates/assembly/assembly.yaml
+++ b/knightos/templates/assembly/assembly.yaml
@@ -26,4 +26,4 @@ files:
     reinit: False
 
 requires: [ 'sass', 'kpack', 'genkfs', 'make' ]
-install: [ 'core/init', 'core/kernel-headers' ]
+install: []
```

But any package that depends on another package (`configlib` or `corelib` depend on eachother), I can't build.

Anyone who used knightos when packages.knightos.org was up should have several manifest.json files within their `~/.cache/knightos/` directory.

**Comment by pixelherodev** [comment]
Wait, packages.knightos.org should be good. I was using it just earlier...

I'll try to look into it shortly. Worst case, I'll get back to you tomorrow
night. If it's down now for you, open a new issue please; PKO is supposed
to be online for the foreseeable future.

**Comment by pixelherodev** [comment]
@earboxer The service appears to have failed; investigating now.

**Comment by pixelherodev** [comment]
This was my fault; after upgrading the PKO server, I didn't upgrade the service. Should be back up within ten minutes.

**Comment by pixelherodev** [comment]
PKO restored.

**Comment by pixelherodev** [comment]
For future notice: PKO *is* expected to work, and if there's a bad gateway again it's a *problem*; feel free to either open an issue here, bring it up in IRC, or email me personally (pleasantatk@gmail.com).
